### PR TITLE
Adding step to clone repository at specific path used by cluster-setup.sh

### PR DESCRIPTION
**Pull request by pshahonica**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This PR ensures that user clone the repository in right path.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

Without cloning repository on defined path, user will have following error
```
./deploy/kubernetes/../common.sh: line 20: /home/user/gopath:/google/gopath/src/sigs.k8s.io/gcp-filestore-csi-driver/deploy/kubernetes/install_kustomize.sh: No such file or directory`
```

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by pshahonica** [comment]
I signed it

File: README.md [hunk 1]
```diff
@@ -90,6 +90,14 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
 
 ## Deploying the Driver
 
+* Clone the repositry in cloudshell using following commands
```
Comment (saikat-royc) [inline README.md:93]: repository

Comment (saikat-royc) [reply README.md:93]: I am merging this change. will fix the typo in my patch

**Pull request merged**